cocos/ui/scroll-view.ts

```diff
@@ -66,7 +66,7 @@ const eventMap = {
     'scroll-to-bottom': 1,
     'scroll-to-left': 2,
     'scroll-to-right': 3,
-    'scrolling': 4,
+    scrolling: 4,
```

Comment:
Why quotes are moved?

Reply:
Fix the es lint error:

Unnecessarily quoted property 'scrolling' found.eslint[quote-props](https://eslint.org/docs/latest/rules/quote-props)